components/ActionSheet/actionSheetAndroid.js

```diff
@@ -124,7 +124,7 @@ export default class ActionSheetAndroid extends Component {
           console.debug('close');
         }}
       >
-        <View style={Style.wrapper}>
+        <View style={[Style.wrapper, this.props.style]}>
```

Comment:
I want to know what kind of style you want to change? the backgroundColor or sth else